InterviewPresenter.swift

```diff
@@ -0,0 +1,51 @@
+
+import UIKit
+import Foundation
+
+protocol ProfileViewInput: AnyObject {
+    var output: ProfileViewOutput? { get set }
+    
+    func setResult(value: String)
+}
+
+protocol ProfileViewOutput {
+    func viewIsReady()
+}
+
+enum TypeCalculate: String {
+    case sum
+    case difference
+}
+
+class Presenter: ProfileViewOutput {
+    weak var view: ProfileViewInput!
```

Comment:
Consider if weak would be sufficient, as well as bear in mind retain cycles. Needs more time to be evaluated.